distribution/src/main/packaging/scripts/postinst

```diff
@@ -52,10 +52,17 @@ case "$1" in
 esac
 
 # to pick up /usr/lib/sysctl.d/elasticsearch.conf
-if command -v /usr/lib/systemd/systemd-sysctl > /dev/null; then
-    /usr/lib/systemd/systemd-sysctl
-elif command -v /lib/systemd/systemd-sysctl > /dev/null; then
-    /lib/systemd/systemd-sysctl
+if [ "${SKIP_SET_KERNEL_PARAMETERS:-false}" == "false" ]; then
+    if command -v /usr/lib/systemd/systemd-sysctl > /dev/null; then
+        /usr/lib/systemd/systemd-sysctl
+    elif command -v /lib/systemd/systemd-sysctl > /dev/null; then
+        /lib/systemd/systemd-sysctl
+    fi
+elif [ "$SKIP_SET_KERNEL_PARAMETERS" == "true" ]; then
+    echo "skipping setting kernel parameters"
+else
+    echo "unrecognized value [$SKIP_SET_KERNEL_PARAMETERS] for SKIP_SET_KERNEL_PARAMETERS; must be unset (defaults to [false]), [false], or [true]"
```

Comment:
The phrase `must be unset (defaults to [false]),` took me a while to parse.  I'd suggest saying this instead: `must be [false] (default)  or [true]`